doc/SUMMARY.md

```diff
@@ -1,24 +1,47 @@
 # Contents
 
   * [Introduction](README.md)
-  * [Getting Started With Gitpod](10_Getting_Started.md)
+  * [Getting Started](10_Getting_Started.md)
   * [Browser Extension](20_Browser_Extension.md)
   * [Workspaces](30_Workspaces.md)
     * [Context URLs](30_Workspaces.md#context-urls)
     * [Life of a Workspace](30_Workspaces.md#life-of-a-workspace)
     * [Shared Workspaces](30_Workspaces.md#shared-workspaces)
   * [Configuring Workspaces](40_Configuration.md)
-    * [`.gitpod` file](40_Configuration.md#gitpod-file)
+    * [`.gitpod` File](40_Configuration.md#gitpod-file)
     * [Docker Image](40_Configuration.md#docker-image)
     * [Exposing Ports](40_Configuration.md#exposing-ports)
     * [Start Script](40_Configuration.md#start-script)
     * [Working with Go](40_Configuration.md#working-with-go)
   * [Theia - Gitpod's IDE](50_IDE.md)
+    * [Overview](50_IDE.md#overview)
     * [Tips & Tricks](52_Tips_and_Tricks.md)
+        * [Command Palette](52_Tips_and_Tricks.md#command-palette)
```

Comment:
Should we really have 3rd level titles in the summary?

Reply:
No, I don't think that's a good idea. Two levels should be sufficient.

Reply:
I thought you might not like it, but it's because the chapters for Theia are actually standalone files, the second-level captions in the left bar are H1s in those chapters, and the third-level ones are H2s. The chapters are long, and if the H2s are not listed anywhere, the page becomes quite hard to navigate. Possible remedy: make a table of contents at the beginning of each chapter (_Search_ chapter already has it.)

Reply:
Related issue: there's nothing to click on to bring the focus to the top of the currently read chapter. The corresponding item in the left bar contains a link to the whole page (not to some artificial anchor placed right in the beginning of it), so because that page is already open, the browser does nothing. Maybe we should fix this, especially if the 3rd-level items in the left bar are replaced with the page contents in the beginning ... so that one can go there with a single click.

Reply:
The navigation behavior is Gitbook's standard. I don't have a strikingly good idea to do it differently. I tried adding a `#` or `#top` to the links in the summary but it didn't work. So I'd prefer to file this as an issue and leave it as is for now.

For the public beta, could we just remove all 3rd level entries from the summary and add the mini table of contents like in search?